src/gui_textual/README.md

```diff
@@ -0,0 +1,38 @@
+# Simple Text-based GUI (TUI) using Textual
+
+*By Andrew Lau*
+
+This example demonstrates using the Python package [Textual](https://textual.textualize.io/)
+to create a simple Text Based Interface to control Zaber devices.
+
+## Hardware Requirements
+Any Zaber linear motion device connected to a serial port.
+
+## Dependencies / Software Requirements / Prerequisites
+The script uses `pipenv` to manage virtual environment and dependencies:
+
+    python3 -m pip install --upgrade pipenv
+
+The dependencies are listed in Pipfile.
+
+## Configuration / Parameters
+The serial port to connect to can be entered into the input box after script startup.
+
+Optinally, you can edit the following constants in the script before running the script:
+- `SERIAL_PORT`: the serial port that your device is connected to.
+For more information on how to identify the serial port,
+see [Find the right serial port name](https://software.zaber.com/motion-library/docs/guides/find_right_port).
+
+## Running the Script
+To run the script:
+
+    cd src/gui_textual
+    pipenv install
+    pipenv run python tui.py
```

Comment:
I know this is probably not the intended way to use this, but I tried running the program from pycharm and it continuously printed a box with "com5" (which I specified in the script) over and over.

When I ran the program from terminal that didn't happen, and the TUI popped up looking normal, but I couldn't get any functions to run. Should you be able to click on the buttons with my mouse? I'm pretty sure I'll be in the office tomorrow, so we should be able to chat then.

Reply:
Added sentence to run this program in a stand-alone terminal instead of the terminal of your IDE.